### PR TITLE
:recycle: Replace Tumblr with Riege

### PR DESCRIPTION
**Pull request by nomaster**

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -4,12 +4,12 @@ Just some random notes about the design of the sidecar injector
 
 ## Mixin Injections
 
-On the note of why the `k8s-sidecar-injector` does not support multiple injections (aka "mixin" injections), https://github.com/tumblr/k8s-sidecar-injector/issues/17 articulates my thought process during design fairly well. Copying the body of the issue for posterity:
+On the note of why the `k8s-sidecar-injector` does not support multiple injections (aka "mixin" injections), https://github.com/riege/k8s-sidecar-injector/issues/17 articulates my thought process during design fairly well. Copying the body of the issue for posterity:
```
Comment (wawe) [inline docs/design.md:7]: Broken link

File: docs/hacking.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ $ make test
 
 ## Image build
 
-The image is build and published on the Hub at https://hub.docker.com/r/tumblr/k8s-sidecar-injector/. See [/docs/deployment.md](/docs/deployment.md) for how to run this in Kubernetes.
+The image is build and published on the Hub at https://hub.docker.com/r/riege/k8s-sidecar-injector/. See [/docs/deployment.md](/docs/deployment.md) for how to run this in Kubernetes.
```
Comment (wawe) [inline docs/hacking.md:25]: Do we actually publish this?

File: README.md [hunk 3]
```diff
@@ -2,36 +2,33 @@
...
+To help solve this, we wrote the `k8s-sidecar-injector`. It is a small service that runs in each Kubernetes cluster, and listens to the Kubernetes API via webhooks. For each pod creation, the injector gets a (mutating admission) webhook, asking whether or not to allow the pod launch, and if allowed, what changes we would like to make to it. For pods that have special annotations on them (i.e. `injector.riege.com/request=logger:v1`), we rewrite the pod configuration to include the containers, volumes, volume mounts, host aliases, init-containers and environment variables defined in the sidecar `logger:v1`'s configuration.
```
Comment (wawe) [inline README.md:9]: We didn't write this initially, maybe we should give credit to Tumblr for most of the implementation here?

**Review by wawe — CHANGES_REQUESTED** [review]

File: docs/hacking.md (hunk 2 above)
Comment (wiebeck) [reply docs/hacking.md:25]: No. Just to our private ACR.

File: docs/design.md (hunk 1 above)
Comment (nomaster) [reply docs/design.md]: I'm deleting the design document, since this is not by us

File: README.md (hunk 3 above)
Comment (nomaster) [reply README.md]: Right, I have added another documentation update.

**Review by wawe — APPROVED** [review]

**Pull request merged**